requirements.txt

```diff
@@ -0,0 +1 @@
+a10-neutron-lbaas
```

Comment:
a10-neutron-lbaas doesn't need to be on the horizon server. a10-neutronclient does.